packages/expo-sqlite/android/src/main/java/expo/modules/sqlite/SQLiteModule.java

```diff
@@ -36,35 +30,30 @@
 
   private static final Map<String, SQLiteDatabase> DATABASES = new HashMap<String, SQLiteDatabase>();
 
-  private ScopedContext mScopedContext;
+  private ModuleRegistry mModuleReqistry;
```

Comment:
```suggestion
  private ModuleRegistry mModuleRegistry;
```

Reply:
BTW I don't see `mModuleRegistry` being used anywhere… maybe we don't need it? 🤔